### DISCUSSION TITLE
Posting Issues without code sandbox

### DISCUSSION THREAD

**Discussion by j-mendez** [vercel/next.js#56258] [Ideas]
### Goals

1. Let people post issues even if the code is not set.
2. Allow getting issues that are important with flexibility.

### Background

Having it set that an issue can only be reported with a link to a codesandbox only works for projects that are public or if theres code that affects changes. Not everything is handled with code or can be reproduced in a manner that someone has code to give. Nextjs is an impressive framework but, bugs arise rapidly across versions.

Theres a couple of issues upon upgrading from `13.4.19` to `13.5` that people are not posting about because of the way issues are blocked. One example is this issue that is ambiguous:

```
// next build
// TypeError: e.replace is not a function
// .next/server/webpack-runtime.js:1:2411
```

Theres no debug information that helps even with inspection, so the ladder is being able to post a valid issue with details on what happened.

### Proposal

Look into removing the codesandbox requirement for posting issues.

**Comment by icyJoseph** [comment]
I think perhaps, the issue template should say, if you don't have a codesandbox, open a discussions instead.

I can speak for myself, but I've helped a bunch of people to, either figure the issue within their setup, or outright narrow down the bug. 

I've also seen this done by others. For example a Stream issue that ended up being traced all the way up to React. It was only possible to really do anything about it, until someone managed to create a reproduction example in the discussions.

For your issue, I guess you'd need to remove UI routes until the build succeeds, then you know you've got the route to investigate. Otherwise the problem ought to be in the root layout, or an Router Handler API route?

Is it cause by the runtime? Edge vs Nodejs?

**Reply by j-mendez** [reply]
Good idea on removing the routes until the issue is found. Only the edge runtime is used.

**Reply by j-mendez** [reply]
The issue I had ended up leading down to this one line of code:

```ts
pdfjs.GlobalWorkerOptions.workerSrc = new URL(
  'pdfjs-dist/build/pdf.worker.min.js',
  import.meta.url
).toString()
```

I think the relevance of having an alternative way for issues should be looked into still.

**Reply by icyJoseph** [reply]
For sure, I think maybe, there should be a new discussion tag, or label, for triage. Like an issue that we can't just pin down at the moment, but we can collect as much data as possible under it.

**Reply by icyJoseph** [reply]
Did you fix your initial issue though?

**Reply by j-mendez** [reply]
@icyJoseph yes, I ended up removing the top level `new URL` and added a require('pdfjs-dist/build/pdf.worker.min.js'). `new URL` seems to not handle the second param `import.meta.url` path replacement correctly on `13.5`.
